### PR TITLE
Ol8 onprem doc

### PR DESCRIPTION
**Pull request by gotsysdba**
Updated doco with helper scripts

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]
please have @rhondaday review too

File: docs-source/spring/content/on-premises/ol8/_index.md [hunk 1]
```diff
@@ -65,30 +98,25 @@ As the `obaas` user, process these steps:
 
    `podman pull container-registry.oracle.com/database/ords:21.4.2-gh`
 
+#### Troubleshooting
+
+If the `podman pull` fails, navigate in a web browser to https://container-registry.oracle.com, click the "database" tile and select "enterprise".  On the right hand side of the page, if prompted, sign-in.  Select "Language" and accept the Terms.  Try the `podman pull` again.
```
Comment (rhondaday) [inline docs-source/spring/content/on-premises/ol8/_index.md:103]: In both instances, add the term "command" after "podman pull"

**Review by rhondaday — COMMENTED** [review]
A few comments.

**Review by corradodebari — APPROVED** [review]

**Review by rhondaday — COMMENTED** [review]
Looks good.

**Pull request merged**